protoc-gen-bq-json-schema/genjson/run.go

```diff
@@ -0,0 +1,25 @@
+package genjson
+
+import (
+	"google.golang.org/protobuf/compiler/protogen"
+	"google.golang.org/protobuf/reflect/protoregistry"
+)
+
+func Run(gen *protogen.Plugin, config Config) error {
+	var files protoregistry.Files
+	for _, file := range gen.Files {
+		if err := files.RegisterFile(file.Desc); err != nil {
+			return err
+		}
+	}
+	for _, f := range gen.Files {
+		if f.Generate {
```

Comment:
I usually do `if !f.Generate` and a `continue` to keep the happy path on a lower indentation level - WDYT?

Reply:
Sure! Changing.